endpoints-support.md

```diff
@@ -135,7 +135,7 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
 |<sub>/rest/firmware-bundles</sub>                                                       | POST     | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |     **Firmware Drivers**                                                                                                                          |
 |<sub>/rest/firmware-drivers</sub>                                                       | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
-|<sub>/rest/firmware-drivers</sub>                                                       | POST     | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/firmware-drivers</sub>                                                       | POST     | :black_circle:       | :black_circle:       |:black_circle:        |
```

Comment:
It was my fault. @adriane-cardozo  checked with me and I did not see that. But it's actually implemented because it inherits the ```create``` method from ```Resource``` class. Sorry guys.

Reply:
no problem, It happens in the best of families. =D
This endpoint has integration tests working well, too.